phpBB/phpbb/db/migration/data/v33x/add_notification_emails_table.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v33x;
+
+class add_notification_emails_table extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return [
+			'\phpbb\db\migration\data\v330\v330',
+		];
+	}
+
+	public function effectively_installed()
```

Comment:
I always thought `effectively_installed()` should precede `depends_on()` :thinking:

Reply:
Too late now 😅

Reply:
Good news :smile_cat: